opencensus/exporters/trace/zipkin/zipkin_exporter.h

```diff
@@ -0,0 +1,132 @@
+// Copyright 2018, OpenCensus Authors
```

Comment:
Please make this file look like e.g. opencensus/exporters/stats/stdout/stdout_exporter.h

Most of this file and the *.cc file should be moved to internal/

Reply:
Done.